### PR TITLE
remove war ops

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Remove the war ops gamemode/challenge from the game and from the game_options.dm

## Why It's Good For The Game
dd1
dd2
dd3
220px-Serbia_Strong

## Changelog
:cl:
del: removal of war-ops gamemode (for now)
/:cl:

### CODE REVIEW

**Comment by Jared-Fogle** [comment]
https://github.com/tgstation/tgstation/pull/52752

https://github.com/tgstation/tgstation/pull/49311

https://github.com/tgstation/tgstation/pull/45307

4th time's the charm

**Comment by itseasytosee** [comment]
Prepare for the tide.

**Comment by Ghilker** [comment]
anger

**Comment by Jared-Fogle** [comment]
Also don't you need to remove the button from the ops shuttle?

**Comment by LemonInTheDark** [comment]
What could go wrong right guys? right?

**Comment by Cartographer-D** [comment]
I hope you have a plan for some alternative, though I do agree it's time to let war ops die.

**Comment by Jared-Fogle** [comment]
> I hope you have a plan for some alternative

Traitors.

**Comment by Cartographer-D** [comment]
Could have a BB variant where they have to nuke the station or have it be a normal traitor obj that's declared. That'd create a lot more paranoia with an internal threat.

**Comment by uomo91** [comment]
https://www.youtube.com/watch?v=LDU_Txk06tM

Next remove wizard gamemode please?

**Comment by itseasytosee** [comment]
Nuke ops aren't dead, only war ops.

**Comment by wesoda25** [comment]
@actioninja please we need you

**Comment by BigGrease** [comment]
wowie we love removing things because some people dont like them

war ops rounds can often be a fun tdm-lite to break up the monotony of rounds where every shift you just do your meta

tho i dont play ops i like fighting ops and i know lots of ppl like being ops and i think it'd be unfair to them to remove this game mode 

fortress station is also fun 

back to back warops is really lame tho so i recommend closing this and simply adding more threat to it to make it less common

**Comment by itseasytosee** [comment]
The only real difference being made here is you can't announce yourself for free credits. You are talking like nukies are being removed from the game.

**Comment by wesoda25** [comment]
Revs and cult already offer something to break up that monotony. Imo the worst issue is the waiting period, you can expect every single war ops round to go the exact same way up until the fighting starts, where a little bit of variety for a select few is offered.

File: code/controllers/configuration/entries/game_options.dm [hunk 1]
```diff
@@ -66,8 +66,6 @@
 
 /datum/config_entry/flag/disable_peaceborg
 
-/datum/config_entry/flag/disable_warops
```
Comment (ATH1909) [inline code/controllers/configuration/entries/game_options.dm:69]: wait, if we already have a config option for this, doesn't that mean that it's the admin team's decision as to whether or not to disable war ops, not ours?

Comment (ATH1909) [reply code/controllers/configuration/entries/game_options.dm:69]: like, if you wanna remove war ops, just ask the headmins to hold a vote for turning off this config option on our servers. there's no need to remove war ops from the code entirely.

Comment (itseasytosee) [reply code/controllers/configuration/entries/game_options.dm:69]: The config has been removed. Rise up against admemes.

**Comment by zxaber** [comment]
War ops is the only thing that makes the Nuke ops mode bearable. Without it, Ops is just a random round reset is triggered by an event most of the station isn't involved with. Just cut the whole thing out.

**Comment by Jared-Fogle** [comment]
> Just cut the whole thing out.

That's the idea, hopefully. My wall of text in the OP describes why I hate ops as a whole, @Rohesie just wants to start here.

**Comment by BarrelOfNapkins** [comment]
Just make war ops less rewarding? 
SS13 is about roleplaying in space, setting up the crew to finally cooperate against a similar foe that they can't just join forces with is one of the most memorable roleplaying experiences out there. Sacrifice yourself to hold off ops that are coming for your captain, save the CMO from a syndicate bomb that just blew up medical because they would know what to do next, hole up in the brig as the last line of defense since the station has no atmosphere, try to surprise bum rush an ops because you've got numbers.
If community can understand why ops is anti-ss13, that may help, otherwise I don't hear any reason why this is good for the game albeit some people don't like it.

**Comment by moocowswag** [comment]
Cult/revs do NOT give off the same vibe as warops. Warops is a unique experience of its own where everyone is united with a common objective. Conversion antags are still incredibly stealth/paranoia based up until they reach a point where they can go loud.

You could make warops give 0 tc and people would still uncommonly use it, not just as a means of achieving something greater than just greentext but as something that structures the round and to give an actual challenge because its not really fun to just go in as 4 people, gang bang the captain, then leave.

This is one of those scenarios where the vast majority of the playerbase wont know what was taken from them until they find out on their own and its too late to voice their opinion.

**Comment by GoldenAlpharex** [comment]
I mean, I dunno, I'd personally just give it a % chance to spawn in the Leader's loadout instead of either being 100% or 0%. I think that'd make things more dynamic, and that it would end up with both the power to go all out on War, but also expecting it to mostly be stealth 90% of the time. It would thus really feel like some rare occurance, instead of just being gone.

**Comment by FatCat919** [comment]
whats the point in playing ss13 if you want everything you dont like removed

**Pull request closed without merging**